packages/amplify-graphql-relational-transformer/src/__tests__/amplify-graphql-has-many-transformer.test.ts

```diff
@@ -3,7 +3,7 @@ import { ModelTransformer } from '@aws-amplify/graphql-model-transformer';
 import { ConflictHandlerType, GraphQLTransform, validateModelSchema } from '@aws-amplify/graphql-transformer-core';
 import { DocumentNode, Kind, parse } from 'graphql';
 import { BelongsToTransformer, HasManyTransformer, HasOneTransformer } from '..';
-import { featureFlags } from './test-helpers';
+import {featureFlags, hasGeneratedField} from './test-helpers';
```

Comment:
nit: format the imports to match the existing style.

Reply:
Done, thought I caught all these but a couple slipped through